option.go

```diff
@@ -56,3 +58,17 @@ func (opt withLogger) config(server *Server) {
 	server.inLogger = opt.in
 	server.outLogger = opt.out
 }
+
+type withVersionInfo struct {
+	versionInfo internal.VersionInfo
+}
+
+func (vi *withVersionInfo) config(server *Server) {
+	server.versionInfo = vi.versionInfo
+}
+
+func WithVersionInfo(info internal.VersionInfo) Option {
```

Comment:
The version info cannot be internal or bridge won't be able to use it (can't import internal packages from outside the module).

You could just make it part of the top level gluon package?

Reply:
Fixed.